bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -241,6 +223,24 @@ pub struct ReplyParameters {
     reply_within_thread: bool,
 }
 
+impl ReplyParameters {
+    fn to_reply(&self) -> Result<Reply, RoomError> {
```

Comment:
Any chance we can make use of a `TryInto` impl here?

Reply:
Ohhh, yes of course! 🤯🤦‍♂️